packages/flutter_tools/lib/src/cmake.dart

```diff
@@ -25,19 +27,66 @@ String _escapeBackslashes(String s) {
   return s.replaceAll(r'\', r'\\');
 }
 
+class _Version {
```

Comment:
What is this giving you that `Version` from `pub_semver` wouldn't do?

Reply:
Done, @stuartmorgan could you take another look and let me know what you think?

Reply:
Marking this as resolved as this comment has been addressed. Let me know if you have additional feedback!